### ISSUE TITLE
Use of plugin introduces false-positive "too few public methods" warning

### ISSUE THREAD

**Issue by claradaia** [jucacrispim/pylint-mongoengine#9]
Hello,

After installing the plugin, I'm getting `R0903: Too few public methods (0/2) (too-few-public-methods)` for Document classes that only describe a schema, some as simple as

```
from mongoengine import Document
from mongoengine.fields import IntField, StringField

class MyModel(Document):
    some_string_field = StringField(required=True)
    some_int_field = IntField(required=True)
```

I do have `min_public_methods=2` in my `.pylintrc`, but without the plugin pylint knows not to warn about these classes. Any idea why?

Best regards.

**Comment by jucacrispim** [comment]
Hi, @claradaia 

Thank you very much for your detailed report.

It is a bug with pylint-mongoengine, in the transformers. The linter can't infer the base mongoengine Document. I'll work on it.

**Comment by jucacrispim** [comment]
I've fixed this on master and released `pylint-mongoengine==0.4.0` to PyPI.
